### PR TITLE
#169259198 User can view the weather for the next five days

### PR DESCRIPTION
**Pull request by RaymondMwaura**
#### What does this PR do?
This pull request adds a 5-day weather forecast component to the application

#### Description of Task to be completed?
- [x] Have the 5-day weather component rendering correctly with the weather forecast details
- [x] Style the component

#### How should this be manually tested?
- After cloning the repo 'cd' into it
- run 'npm install' to install all the dependencies
- run 'npm start' to start the server
You should see the weather-app loaded on the browser with the 5-day weather forecast rendered with the weather data

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
#169259198

#### Screenshots (if appropriate)
5dayweather

#### Questions:
N/A

### CODE REVIEW

**Comment by b0nbon1** [comment]
please merge the commit that you pick from my changes into your commit so as to avoid conflicts.

**Comment by Acrunam** [comment]
> please merge the commit that you pick from my changes into your commit so as to avoid conflicts.

I have now merged them.

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+#### What does this PR do?
+#### Description of Task to be completed?
+#### How should this be manually tested?
+#### Any background context you want to provide?
+#### What are the relevant pivotal tracker stories?
+#### Screenshots (if appropriate)
+#### Questions:
```
Comment (b0nbon1) [inline .github/pull_request_template.md]: this should be in separate PR so remove it, its part of this PR

File: src/App.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    };
+  }
+
+  componentDidMount() {
+    const API = `http://api.openweathermap.org/data/2.5/forecast?q=${this.state.city}&appid=047ef33a4d9c38d3dddaa4b631c96d45`;
+    // this.setState({ isLoading: true });
+    fetch(API)
+      .then((response) => response.json())
+      .then((data) => this.setState({ weather: data, isLoading: false }));
+  }
```
Comment (b0nbon1) [inline src/App.js]: keep up with the changes from the app component

Comment (Acrunam) [reply src/App.js]: These are now updated.

File: .github/pull_request_template.md (hunk 1 above)
Comment (Acrunam) [reply .github/pull_request_template.md]: I have removed the file from this pull request.
